backend/src/main/java/disaster/model/mappers/DisasterAlertHazardEventDeserializer.java

```diff
@@ -35,4 +36,24 @@ public HazardEventApiDto deserialize(JsonParser p, DeserializationContext ctx) t
         hazardEvent.setLatitude(latitude);
         return hazardEvent;
     }
+
+    private HazardEventType getHazardEventType(String string) {
+        if (string.equals("FLOOD"))
```

Comment:
https://stackoverflow.com/questions/604424/how-to-get-an-enum-value-from-a-string-value-in-java
try valueOf and return UNKNOWN on error?